readme.md

```diff
@@ -11,34 +11,68 @@
 ## Install
 
 ```bash
-npm install --save react-fontawesome
+$ npm install --save react-fontawesome
```

Comment:
Can you remove the `$` prefixes, they make the code not copy-pastable and may not be clear to some users that they indicate "do this on the command line"